README.md

```diff
@@ -1,24 +1,96 @@
-# README
+# BudgetApp.
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> This application tracks a user's purchases across many needs and wants categories.It enables users to maintain categories, add or remove transactions within specific categories, and keep track of all transactions under a particular category.
 
-Things you may want to cover:
+## Built With
 
-* Ruby version
+- Ruby
+- Ruby on Rails v7.0.4
+- Gems used for testing: Capybara, Rspec-Rails, and Shoulda-matchers.
 
-* System dependencies
+## Live Link
+ https://deliciouscash.herokuapp.com
+ 
+## Demo Link
+ https://www.loom.com/share/7516f94388dc4416af02484322b8b0cf
 
-* Configuration
+## Getting Started
 
-* Database creation
+To get a local copy up and running follow these steps:
 
-* Database initialization
+### Prerequisites
 
-* How to run the test suite
+- Ruby
+- Rails: 7.0.4
+- Postgres
+- Git
 
-* Services (job queues, cache servers, search engines, etc.)
+### Usage
 
-* Deployment instructions
+- Fork/Clone this project to your local machine
+- Open folder in your local enviroment and run thes lines of code to get started:
 
-* ...
+Install gems with:
+
+```Ruby
+    bundle install
+```
+
+Setup database with:
+
+```Ruby
+   rails db:create
+   rails db:migrate
+   rails db:seed
+```
+
+Start server with:
+
+```Ruby
+    rails s -p 3001
+```
+
+Then open a web page and go to [port 3001 on your local machine.](http://localhost:3001)
+
+## Running tests
+
+```Ruby
+    bundle exec rpsec
+```
+
+## Deployment
+
+Follow vendor specific instructions to deploy the application.
+
+## Authors
+
+👤 **Assad Isah**
+
+- Github: [nottherealalanturing](https://github.com/nottherealalanturing)
+- LinkedIn: [Assad Isah](https://linkedin.com/in/assadisah)
+
+## 🤝 Contributing
+
+Contributions and feature requests are welcome!
+
+Start by:
+
+- Forking the project
+- Cloning the project to your local machine
+- `cd` into the project directory
+- Run `git checkout -b your-branch-name`
+- Make your contributions
+- Push your branch up to your forked repository
+- Open a Pull Request with a detailed description to the development(or master if not available) branch of the original project for a review
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
```

Comment:
Please make sure the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).